node/src/shared/store/actions/products.js

```diff
@@ -0,0 +1,36 @@
+import {FETCH_PRODUCTS_SUCCESS, FETCH_PRODUCT_SUCCESS} from "./actionsTypes"
```

Comment:
shared  folder is used for components that are used in multiple places. It does not seem it to be one of this kind

Reply:
Done